packages/pulp/pulp.spec

```diff
@@ -2,7 +2,7 @@
 %{!?python_sitearch: %global python_sitearch %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
 
 # don't use <= 6 here, since %{?rhel} is empty on fedora, and 0 is <= 6
-%if 0%{?rhel} == 5 || 0%{?rhel} == 6
+%if 0%{?rhel} == 5 || 0%{?rhel} == 6 || 0%{?suse_version}
```

Comment:
Does that mean only the pulp consumer is built on SUSE?

Reply:
Only the client part is required to use the katello-agent on SUSE.